jscomp/core/lam_convert.ml

```diff
@@ -505,9 +505,13 @@ let convert (exports : Set_ident.t) (lam : Lambda.lambda) :
                  primitive %s"
                 s
         in
-        let args = Ext_list.map args convert_aux in
+        let dynamic_import = match primitive with
```

Comment:
Can be simplified as `primitive = Pimport`.

Reply:
Thanks! Done https://github.com/rescript-lang/rescript-compiler/pull/6232/commits/36038524bb53874a2299e3993ec6b4cc5e244965